docs/reference/custom resources/verifiers.md

```diff
@@ -75,15 +79,19 @@ spec:
 apiVersion: config.ratify.deislabs.io/v1beta1
 kind: Verifier
 metadata:
-  name: verifier-notation
+  name: notation-wabbit
```

Comment:
curious why we rename it

Reply:
Just want to ensure align with the sample config in ratify code base